### PR TITLE
feat(components/notifications): add documentation for global alert

### PR DESCRIPTION
**Pull request by Francois-Esquire**
## Description

Adds documentation for the new `GlobalAlert` component.

## Detail

"Types" example preview.

Screen Shot 2022-11-15 at 4 47 21 PM

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [x] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/global-alert.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import Types from '../../examples/notifications/global-alert/GlobalAlertTypes.tsx';
+import TypesCode from '!!raw-loader!../../examples/notifications/global-alert/GlobalAlertTypes.tsx';
+
+<CodeExample code={TypesCode}>
+  <Types />
+</CodeExample>
+
+## How to use it well
+
+### Clearly actionable
```
Comment (jzempel) [inline src/pages/components/global-alert.mdx:61]: This section should have an associated example, as it is imperative for developers to see how to properly use anchors.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/pages/components/global-alert.mdx (hunk 1 above)
Comment (Francois-Esquire) [reply src/pages/components/global-alert.mdx:64]: @jzempel agreed - FYI, we added a variant with a link example.

File: src/pages/components/global-alert.mdx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Allow for one action button for the user to address the issue
+associated with the banner, with the exception of “learn more”.
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/pages/components/global-alert.mdx:67]: @jzempel yeah for sure, could you give a restructured example?

File: src/examples/notifications/global-alert/GlobalAlertTypes.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import { Anchor } from '@zendeskgarden/react-buttons';
+
+const StyledSpacer = styled.div`
+  height: ${p => p.theme.space.md};
+`;
+
+const Example = () => (
+  <>
+    <GlobalAlert type="info">
+      <GlobalAlert.Title>New update available</GlobalAlert.Title>
```
Comment (jzempel) [inline src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: This example is highlighting a styling fault with `GlobalAlert`. On first glance, I think there is a missing `flex-shrink: 0` on this element. Please note for a follow-on component fix.

File: src/examples/notifications/global-alert/GlobalAlertTypes.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      <GlobalAlert.Content>
+        <span>Your account will automatically update in 5 days. </span>
+        <Anchor href="#" isExternal>
+          Find out more
+        </Anchor>
```
Comment (jzempel) [inline src/examples/notifications/global-alert/GlobalAlertTypes.tsx:25]:
```suggestion
        Your account will automatically update in 5 days.{' '}
        <Anchor href="#" isExternal>
          Find out more
        </Anchor>
```

Prefer simplified DOM structure.

File: src/examples/notifications/global-alert/GlobalAlertTypes.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    <StyledSpacer />
+
+    <GlobalAlert type="warning">
+      <GlobalAlert.Title>Warning</GlobalAlert.Title>
+      <GlobalAlert.Content>
+        <span>During system maintenance, you may experience a service disruption. </span>
+        <Anchor href="#" isExternal>
+          Learn more
+        </Anchor>
```
Comment (jzempel) [inline src/examples/notifications/global-alert/GlobalAlertTypes.tsx:39]:
```suggestion
        During maintenance, you may experience a service disruption.{' '}
        <Anchor href="#" isExternal>
          Learn more
        </Anchor>
```

Suggested word removal so example lays out better with default page width without odd external icon wrap.

File: src/pages/components/global-alert.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jzempel) [reply src/pages/components/global-alert.mdx:67]: We can use https://hemingwayapp.com/ to button this up. But I'm having a difficult time understanding what it actually means.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/examples/notifications/global-alert/GlobalAlertTypes.tsx (hunk 3 above)
Comment (Francois-Esquire) [reply src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: I'll create a PR to fix this.

Comment (smritimadan) [reply src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: @Francois-Esquire Can you pls create a ticket for this? thx!

File: gatsby-config.js [hunk 6]
```diff
@@ -27,6 +27,9 @@ const figmaNodeIds = [
   '554:9877' /* components-dropdowns-combobox-searchable-dont */,
   '108:4872' /* components-forms-checkbox-standalone-do */,
   '108:4897' /* components-forms-checkbox-standalone-option-dont */,
+  '1459:39467' /* components-notification-global-alert-clearly-actionable */,
+  '1414:38729' /* components-notification-global-alert-dismissable */,
+  '1414:38800' /* components-notification-global-alert-persistent */,
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply gatsby-config.js:32]: This was a mistake on my end, I removed the `imageIsSquare` prop from the `Do` component and the images should be displaying properly now.

File: src/examples/notifications/global-alert/GlobalAlertTypes.tsx (hunk 3 above)
Comment (Francois-Esquire) [reply src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: @smritimadan for sure, I'll do that!

Comment (Francois-Esquire) [reply src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: Raised a PR in the meantime, [`fix(notifications): add flex-shrink to global alert title`)](https://github.com/zendeskgarden/react-components/pull/1464).

Comment (jzempel) [reply src/examples/notifications/global-alert/GlobalAlertTypes.tsx:20]: Actually @Francois-Esquire, I think the structure is incorrect here. Based on late game component discussions, the `GlobalAlert.Title` is meant to live _within_ `GlobalAlert.Content`. I don't think there is a need to update the component. Rather, these examples should be restructured.

**Review by jzempel — APPROVED** [review]
Good on my end, pending https://github.com/zendeskgarden/website/pull/446/files#r1035070025 check from @steue.

File: src/pages/components/global-alert.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/pages/components/global-alert.mdx:66]:
```suggestion
Provide one button to address the issue, and an optional way to “learn more”.
```

**Review by steue — APPROVED** [review]
this looks great. thank you @Francois-Esquire and @jzempel for making this happen!

**Pull request merged**